src/components/joblogger/JobLogger.js

```diff
@@ -194,7 +194,13 @@ const JobLogger = ({ visible, onClose, onStatusChange }) => {
           ))}
         </Stack>
       </Stack>
-      <Box flex={1} opacity={0.5} backgroundColor="black" />
+      <Box
+        flex={1}
+        opacity={0.5}
+        backgroundColor="black"
+        onClick={onClose}
+        cursor="pointer"
```

Comment:
Not sure if we need to change the cursor for this. This also doesn't happen when you click outside a modal